_analyzers/token-filters/lowercase.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Lowercase
+parent: Token filters
+nav_order: 260
+---
+
+# Lowercase token filter
+
+The `lowercase` token filter is used to convert all characters in the token stream to lowercase, making searches case insensitive.
+
+## Parameters
+
+The `lowercase` token filter can be configured with the following parameter.
+
+Parameter | Required/Optional | Description
+:--- | :--- | :---
+ `language` | Optional | Specifies a language-specific token filter to use for lowercasing. Valid values are: <br>- [`greek`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/el/GreekLowerCaseFilter.html) <br>-  [`irish`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/ga/IrishLowerCaseFilter.html) <br>-  [`turkish`](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/tr/TurkishLowerCaseFilter.html). <br> Default is [Lucene’s LowerCaseFilter](https://lucene.apache.org/core/8_7_0/analyzers-common/org/apache/lucene/analysis/core/LowerCaseFilter.html). 
+
```

Comment:
Line 18: Is `english` also an option, or is that the default used by the Lucene filter?

Reply:
Default without specifying a language.